addons/website/views/snippets/s_carousel_cards.xml

```diff
@@ -0,0 +1,190 @@
+                            <span class="carousel-control-prev-icon" aria-hidden="true"/>
+                            <span class="visually-hidden">Previous</span>
+                        </button>
+                        <button class="carousel-control-next bottom-0 o_not_editable" contenteditable="false" t-attf-data-bs-target="#myCarouselCards{{uniq}}" data-bs-slide="next" role="img" aria-label="Next" title="Next">
+                            <span class="carousel-control-next-icon" aria-hidden="true"/>
+                            <span class="visually-hidden">Next</span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="snippet_options_carousel_cards" inherit_id="website.snippet_options_carousel_bottom_controllers" primary="True">
+    <xpath expr="//div" position="attributes">
+        <attribute name="data-target">.s_carousel_cards</attribute>
+        <attribute name="data-exclude"/>
+        <attribute name="data-js">CarouselCards</attribute>
+    </xpath>
+    <xpath expr="(//we-select)[3]" position="after">
+        <we-checkbox string="Images"
+            data-select-class="s_carousel_cards_with_img"
+            data-toggle-card-img=""
+            data-no-preview="true"
+            data-name="toggle_card_img_opt"/>
+        <we-button-group string="Position"
+            class="o_we_sublevel_1"
+            data-apply-to=".s_card"
+            data-dependencies="toggle_card_img_opt">
+            <we-button title="Left"
+                data-select-class="flex-lg-row"
+                data-img="/website/static/src/img/snippets_options/pos_left.svg"/>
+            <we-button title="Right"
+                data-select-class="flex-lg-row-reverse"
+                data-img="/website/static/src/img/snippets_options/pos_right.svg"/>
+        </we-button-group>
+
+        <t t-set="colSize" t-value="10"/>
+        <t t-set="ratioSize" t-value=".1"/>
+        <we-range string="Width"
+            class="o_we_sublevel_1"
+            data-dependencies="toggle_card_img_opt"
+            title="Adjust the image width"
+            data-select-style="0"
+            data-css-property="--card-img-size-h"
+            data-unit="%"
+            t-att-data-min="colSize"
+            t-att-data-max="colSize * 7"
+            t-att-data-step="colSize"/>
+        <we-range string="Height"
+            class="o_we_sublevel_1"
+            data-dependencies="toggle_card_img_opt"
+            title="Adjust the image ratio"
+            data-select-style="0"
+            data-css-property="--card-img-ratio-v"
+            data-unit=""
+            t-att-data-min="ratioSize"
+            t-att-data-max="ratioSize * 10"
+            t-att-data-step="ratioSize"/>
```

Comment:
We can probably leave this to the task about unifying carousel options @lebl-odoo, maybe you could take note: this new snippet has things that could be unified too. Like this height option, which is a duplicate of the height option of image gallery, presented another way.

(you tested this new snippet?)

I find this height slider a bit weird, sometimes it has no effect, probably because the image has a min-height or the card itself? But then, should not we just not suggest those values?

Reply:
Thx for the ping @qsm-odoo , there are several new sliders snippets. I'll have a look to standardize those options once at the end.

> Like this height option, which is a duplicate of the height option of image gallery, presented another way.

Didn't pay attention, thx to point it out.

For reference:

| s_image_gallery | s_carousel_cards |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/c614bff7-12e8-4c0a-bbad-3203b90ffc0b) | ![image](https://github.com/user-attachments/assets/e47b30bf-9f6c-4b90-b410-543d04eb84c0) | 

> I find this height slider a bit weird, sometimes it has no effect, probably because the image has a min-height or the card itself? But then, should not we just not suggest those values?

Good catch! Moreover, by playing a bit this morning, I was able to remove the indicators. Then I got a traceback. It's not clear how to reproduce it, but I did (typing multiple times on delete and backspace) https://drive.google.com/file/d/18R59A-jMLBiATlhIMmV-hh1JLG5YEc7p/view

Reply:
Hey @lebl-odoo, thanks for the test 👍
I tried several times to reproduce this issue but I couldn't 😕

Maybe @qsm-odoo would have an idea about this?

Reply:
> Did you adapt based on that @Brieuc-brd ?

Thanks to @stefanorigano, we've reworked this feature.
Rather than modifying the image aspect ratio, we now add an additional height, based on the bottom padding of the card body.
This feature is now independent of the image height, which avoids the odd behavior that sometimes prevented changes when moving the input. I've also added the value next to the input to make this feature easier to use.

![Capture d’écran 2024-12-09 à 09 26 31](https://github.com/user-attachments/assets/6d0fbe47-ecb8-4116-8f7b-74abf083f8a9)


What do you think @qsm-odoo @lebl-odoo ?